src/main/java/ch/njol/skript/effects/EffSendResourcePack.java

```diff
@@ -1,115 +0,0 @@
-/**
- *   This file is part of Skript.
- *
- *  Skript is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
- *
- *  Skript is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
- *
- * Copyright Peter Güttinger, SkriptLang team and contributors
- */
-package ch.njol.skript.effects;
-
-import org.bukkit.entity.Player;
-import org.bukkit.event.Event;
-import org.jetbrains.annotations.Nullable;
-
-import ch.njol.skript.Skript;
-import ch.njol.skript.doc.Description;
-import ch.njol.skript.doc.Examples;
-import ch.njol.skript.doc.Name;
-import ch.njol.skript.doc.Since;
-import ch.njol.skript.lang.Effect;
-import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser.ParseResult;
-import ch.njol.util.Kleenean;
-import ch.njol.util.StringUtils;
-
-@Name("Send Resource Pack")
-@Description({"Request that the player's client download and switch resource packs. The client will download ",
-		"the resource pack in the background, and will automatically switch to it once the download is complete. ",
-		"The URL must be a direct download link.",
-		"",
-		"The hash is used for caching, the player won't have to re-download the resource pack that way. ",
-		"The hash must be SHA-1, you can get SHA-1 hash of your resource pack using ",
-		"<a href=\"https://emn178.github.io/online-tools/sha1_checksum.html\">this online tool</a>.",
-		"",
-		"The <a href='events.html#resource_pack_request_action'>resource pack request action</a> can be used to check ",
-		"status of the sent resource pack request."})
-@Examples({"on join:",
-		"	send the resource pack from \"URL\" with hash \"hash\" to the player"})
-@Since("2.4")
-public class EffSendResourcePack extends Effect {
-
-	static {
```

Comment:
Personally, I dont think it's that big of a drastic change. But idk, guess we can wait and see

Reply:
I don't see much point in deleting it, take the spawn section for example, theres an effect & a section

Reply:
Yeah, this seems like a breaking change for no reason in my eyes too. Even if you delete the class, why not replicate the behaviour of the old way in the new EffSec class?

Reply:
You can just keep it the same, but add the rest of the features in the section, gives people more reason to use your sectio n anyway if it has an option they may be looking for

Reply:
So you want me to make the pattern exactly the same as the `EffSendResourcePack`? including the  `with hash`?

Reply:
no i meant bring back the resource pack effect exactly how it was, (without the features your section is providing)

Reply:
People who update to a newer version shouldnt suddently have errors popping up. The old behaviour, ideally, should still be supported, and your new section should just be an optional thing for users

Reply:
You can still do it regularly with the EffSec, it doesnt need to be a section, it will still work without being a section
`send the resource pack from the URL "string" to players`

Reply:
But the previous effect offered more options than that didnt it?

Reply:
Done